common/src/app/common/pages/changes.cljc

```diff
@@ -15,6 +15,20 @@
    [app.common.pages.spec :as spec]
    [app.common.spec :as us]))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
Aren't those abstract enough to be in common/data?

Reply:
Yes and no, right now this is only useful in this ns, until now we didn't needed it, so I prefer keeping them here until we really need as general purpose.

An other reason is because we already have without-nils that can be changed to be polymorphic, but in this case want to ensure that it always returns a vector and not a coll of the same type as the input.